hw03_hilla3.py

```diff
@@ -0,0 +1,53 @@
+# TODO Do not edit this file directly! Instead, create a new file called
+# TODO a03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Andy Hill
+# Username: hilla3
+#
+# Assignment: HW03
+# Purpose: to learn how to push and pull
+# Google Doc Link:
+#https://docs.google.com/document/d/1kNaw8-ZA9qom9Ylts-O4CE4vvRIhTbGhVIZIhzmSbD4/edit
+#################################################################################
+# Acknowledgements:
+#
+#################################################################################
+
+import turtle
+
+cas = turtle.Turtle()
```

Comment:
There should be no code outside at the highest level (no indentation). This should be in `main()`, and when you need access to the turtle in the functions, you pass it into them:

```
def cassy(cas):
   ...

def main():
    cas = turtle.Turtle()
    cassy(cas)
```